blog/h-language-2019-06-30.markdown

```diff
@@ -0,0 +1,325 @@
+---
+title: The h Programming Language
+date: 2019-06-30
+---
+
+# The h Programming Language
+
+[h](https://h.christine.website) is a project of mine that I have released 
+recently. It is a single-paradigm, multi-tenant friendly, turing-incomplete
+programming language that does nothing but print one of two things:
+
+- the letter h
+- a single quote (the Lojbanic "h")
+
+It does this via [WebAssembly](https://webassembly.org). This may sound like a
+pointless complication, but actually this ends up making things _a lot simpler_.
+WebAssembly is a virtual machine (fake computer that only exists in code) intended
+for browsers, but I've been using it for server-side tasks.
+
+I have written more about/with WebAssembly in the past in these posts:
+
+- https://christine.website/talks/webassembly-on-the-server-system-calls-2019-05-31
+- https://christine.website/blog/olin-1-why-09-1-2018
+- https://christine.website/blog/olin-2-the-future-09-5-2018
+- https://christine.website/blog/land-1-syscalls-file-io-2018-06-18
+- https://christine.website/blog/templeos-2-god-the-rng-2019-05-30
+
+This is a continuation of the following two posts:
+
+- https://christine.website/blog/the-origin-of-h-2015-12-14
+- https://christine.website/blog/formal-grammar-of-h-2019-05-19
+
+All of the relevant code for h is [here](https://github.com/Xe/x/tree/master/cmd/h).
```

Comment:
If you don't mind me asking here, has the code moved somewhere else?

I found commit https://github.com/Xe/x/commit/41bb9eb9d3881bfb48bb3077a0d83b802a5bb1ab which suggests it moved to another repository, but doesn't say where. Is it available somewhere at this time? (It's no problem if not.)

Thank you.

Reply:
See here: https://github.com/Xe/hlang

I would link you to the copy on my gitea server but the kubernetes gods have said that it shouldn't be up.